### ISSUE TITLE
You are entitled to wear the Humane Tech badge

### ISSUE THREAD

**Issue by aschrijver** [Widdershin/programmers-oath#114]
Hi @Widdershin ,

I am community moderator and [forum](https://community.humanetech.com) admin of [The Center for Humane Technology](https://humanetech.com), and - as I mentioned in another issue - I have added this repository to the curated [awesome humane tech](https://raw.githubusercontent.com/engagingspaces/awesome-humane-tech) list.

This means that you are entitled to display the [![Humane Tech](https://raw.githubusercontent.com/engagingspaces/awesome-humane-tech/master/humane-tech-badge.svg?sanitize=true)](https://github.com/engagingspaces/awesome-humane-tech) badge on your README.

If you would like that, of course, otherwise.. no offence taken :slightly_smiling_face: 

You can add it after or directly below the title, or alternatively in a section at the bottom of the page (maybe with some more text to it), by adding:

```markdown
[![Humane Tech](https://raw.githubusercontent.com/engagingspaces/awesome-humane-tech/master/humane-tech-badge.svg?sanitize=true)](https://github.com/engagingspaces/awesome-humane-tech)
```

I can PR if you prefer that.

**Comment by Widdershin** [comment]
That's a lovely offer, thank you. I would definitely merge a PR adding this, my preference would be a section at the bottom with some more text to it.

Also, just wanted to say that I've really enjoyed reading the Humane Tech forum so far, and that it's inspired me to consider a lot of new issues. You're doing great work 😄

**Comment by aschrijver** [comment]
Oww.. I am so sorry @Widdershin but this issue has slipped my radar :(

Thank you, I am glad to hear that you enjoy the forum so much! They are important subjects being broached there. Unfortunately I haven't been participating as much as I used to in the past couple of weeks, but I have a whole batch of stuff that I _should_ post there really.

I have created a PR on the Readme with a proposal, and we can fine-tune it from there.
